build.py

```diff
@@ -65,12 +65,12 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.12.0dev':
-      ('21.07dev',   # triton container
-       '21.06',      # upstream container
-       '1.8.0',      # ORT
-       '2021.2.200', # ORT OpenVINO
-       '2021.2')     # Standalone OpenVINO
+    '2.12.0dev': (
```

Comment:
format fixes

Reply:
This is with the `format.py` formatting...